### PR TITLE
{Servicebus/Eventhub} Deprecation Alert for `az servicebus migration update` `az eventhubs georecovery-alias update` `az servicebus georecovery-alias update` cmdlets

### PR DESCRIPTION
**Pull request by schaudhari6254888**
… update` `az eventhubs georecovery-alias update` `az servicebus georecovery-alias update` cmdlets

**Related command**

**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Servicebus/Eventhub

File: src/azure-cli/azure/cli/command_modules/servicebus/commands.py [hunk 1]
```diff
@@ -60,3 +63,6 @@ def load_command_table(self, _):
     with self.command_group('servicebus namespace encryption', custom_command_type=sb_namespace_custom, is_preview=True) as g:
         g.custom_command('add', 'cli_add_encryption')
         g.custom_command('remove', 'cli_remove_encryption')
+
+# Migration Region
+    self.command_table['servicebus migration update'] = Update(loader=self, deprecate_info=self.deprecate())
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/servicebus/commands.py:68]: I don't see a new import, so this `Update` is the same with which was imported from line 43, right?
```
from azure.cli.command_modules.servicebus.aaz.latest.servicebus.georecovery_alias._update import Update
```

File: src/azure-cli/azure/cli/command_modules/eventhubs/commands.py [hunk 2]
```diff
@@ -50,6 +50,8 @@ def load_command_table(self, _):
     with self.command_group('eventhubs georecovery-alias', custom_command_type=eh_namespace_custom,
                             is_preview=True) as g:
         g.custom_command('set', 'set_georecovery_alias', supports_no_wait=True)
+    from azure.cli.command_modules.eventhubs.aaz.latest.eventhubs.eventhub._update import Update
+    self.command_table['eventhubs georecovery-alias update'] = Update(loader=self, deprecate_info=self.deprecate())
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/eventhubs/commands.py:54]: This is the first time I saw manual deprecation based on aaz commands. @kairu-ms Is this the standard way?

Comment (kairu-ms) [reply src/azure-cli/azure/cli/command_modules/eventhubs/commands.py:54]: Yes. That's the standard way.

File: src/azure-cli/azure/cli/command_modules/servicebus/commands.py (hunk 1 above)
Comment (schaudhari6254888) [reply src/azure-cli/azure/cli/command_modules/servicebus/commands.py:69]: Yeah, I forgot this is a different import.
Corrected now.
Thanks!

File: src/azure-cli/azure/cli/command_modules/servicebus/commands.py [hunk 3]
```diff
@@ -6,6 +6,7 @@
 # pylint: disable=line-too-long
 # pylint: disable=too-many-statements
 # pylint: disable=too-many-locals
+# pylint: disable=all
```
Comment (evelyn-ys) [inline src/azure-cli/azure/cli/command_modules/servicebus/commands.py:9]: Pls don't disable all pylint............

You can add specific pylint disable rule in specific line

Comment (schaudhari6254888) [reply src/azure-cli/azure/cli/command_modules/servicebus/commands.py]: corrected!

**Review by evelyn-ys — APPROVED** [review]

**Pull request merged**